### ISSUE TITLE
Sugestion: activities could list the creation of users

### ISSUE THREAD

**Issue by flashfs** [RestyaPlatform/board#1243]
This is just a suggestion of enhancement. Activities list on administrator panel could also list the creation of users (manual or from LDAP).

**Comment by rrjanbiah** [comment]
@flashfs Are you referring to the "Add User" that's available in admin users page? (refer screenshot below)

![rb-admin-add-user](https://user-images.githubusercontent.com/4907427/28167724-2ef08696-67fa-11e7-834b-8c08ccbe835e.png)

**Comment by flashfs** [comment]
No, the Activities menu. I can see the deletion of users, but not the creation (manually or automatically from LDAP).
![activities](https://user-images.githubusercontent.com/2033838/28168314-104d9c96-67b5-11e7-8b0c-17a32d1161fc.jpg)
